mod/item.php

```diff
@@ -175,6 +175,19 @@ function item_post(&$a) {
 	$app         = ((x($_REQUEST,'source'))      ? strip_tags($_REQUEST['source'])  : '');
 	$extid       = ((x($_REQUEST,'extid'))       ? strip_tags($_REQUEST['extid'])  : '');
 
+	// Check for multiple posts with the same message id (when the post was created via API)
+	if (($message_id != "") AND ($profile_uid != 0)) {
```

Comment:
Currently it is completely mixed in the code. Parts that were written by me mostly contain the ". I hadn't looked into the PSR for this. I guess I have to do it.